common/components/WalletDecrypt/components/LedgerNano.tsx

```diff
@@ -22,11 +28,13 @@ interface State {
   showTip: boolean;
 }
 
-export class LedgerNanoSDecrypt extends Component<Props, State> {
+type Props = OwnProps & StateProps;
+
+class LedgerNanoSDecryptClass extends Component<Props, State> {
   public state: State = {
     publicKey: '',
     chainCode: '',
-    dPath: DEFAULT_PATH,
+    dPath: this.props.dPath.value,
```

Comment:
Can you do this? I thought this had to happen in constructor.

Reply:
I'll need to check back, but during testing it works fine. It might however be because some state is being updated immediately after mount, so I can't be certain.